<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrainAmpSeries Connector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Channel Labels</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="channelLabels">
         <property name="plainText">
          <string>Fp1
Fp2
F7
F3
Fz
F4
F8
FC5
FC1
FC2
FC6
T7
C3
Cz
C4
T8
TP9
CP5
CP1
CP2
CP6
TP10
P7
P3
Pz
P4
P8
PO9
O1
Oz
O2
PO10 </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Device Settings</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Device Number</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="deviceNumber">
           <property name="toolTip">
            <string>The number of the USB device (if multiple); the first one is #1.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number of Channels</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="channelCount">
           <property name="toolTip">
            <string>This must match the number of entries in the channel list</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>256</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Impedance Mode</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="impedanceMode">
           <property name="toolTip">
            <string>The default setting is to operate in high-impedance mode (less need for perfect electrode contact)</string>
           </property>
           <item>
            <property name="text">
             <string>High</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Low</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="resolution">
           <property name="toolTip">
            <string>Resolution of the measured signal</string>
           </property>
           <item>
            <property name="text">
             <string>100 nV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500 nV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 µV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>152.6 µV</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>DC Coupling</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="dcCoupling">
           <property name="toolTip">
            <string>The default is AC</string>
           </property>
           <item>
            <property name="text">
             <string>AC</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DC</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Chunk Size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="chunkSize">
           <property name="toolTip">
            <string>The number of samples per chunk emitted by the driver -- a small value will lead to lower overall latency but causes more CPU load</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Use PolyBox</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="usePolyBox">
           <property name="toolTip">
            <string>If this is checked then the first 8 channels will hold the polybox signals; make sure to increase your channel count appropriately.</string>
           </property>
           <property name="text">
            <string>(check)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>LSL Trigger Output Style</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Unsampled String Markers</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="unsampledMarkers">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For an explanation of trigger marker types please read 'explanation_of_trigger_marker_types.pdf'.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>(check)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Sampled String Markers</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="sampledMarkers">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For an explanation of trigger marker types please read 'explanation_of_trigger_marker_types.pdf'.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>(check)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Floating Point EEG Channel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="sampledMarkersEEG">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For an explanation of trigger marker types please read 'explanation_of_trigger_marker_types.pdf'.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>(check)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="linkButton">
          <property name="text">
           <string>Link</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Configuration"/>
    <addaction name="actionSave_Configuration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
